behave/configuration.py

```diff
@@ -452,14 +497,82 @@ def read_configuration(path):
     return result
 
 
+def read_toml(path):
+    """Read configuration from pyproject.toml file.
+
+    Configuration should be stored inside the 'tool.behave' table.
+
+    See https://www.python.org/dev/peps/pep-0518/#tool-table
+    """
+    # pylint: disable=too-many-locals, too-many-branches
+    with open(path, "rb") as f:
```

Comment:
Why `"rb"`? Shouldn't we use read in text mode for cross-platform compatibility?

```suggestion
    with open(path) as tomlfile:
```

We shouldn't trust that configuration readers can handle Windows and Unix newlines interchangeably.

Pylint suggests to use self-explanatory variable names.